groupnews-service/src/test/java/com/epam/internship/UserServiceTest.java

```diff
@@ -74,20 +75,31 @@ public void shoudlReturnAllUsers() {
 
 	@Test
 	public void shouldCreateUser() {
-		//When
+		// When
 		User user = systemUnderTest.createUser(USER_NAME_1, USER_EMAIL_1);
-		//Then
+		// Then
 		assertEquals(USER_NAME_1, user.getName());
 		assertEquals(USER_EMAIL_1, user.getEmail());
 	}
-	
+
 	@Test
-	public void shouldReturnFalseWithUniqueEmail(){
+	public void shouldReturnFalseWithUniqueEmail() {
 		assertFalse(systemUnderTest.emailAlreadyExists(USER_UNIQUE_EMAIL));
 	}
-	
+
 	@Test
-	public void shouldReturnFalseWithExistingEmail(){
+	public void shouldReturnFalseWithExistingEmail() {
 		assertTrue(systemUnderTest.emailAlreadyExists(USER_EMAIL_1));
 	}
+
+	@Test
+	public void shouldReturnUserForMatchingId() {
+		// Given
+		Mockito.when(systemUnderTest.getUserById(USER_ID_1)).thenReturn(user1);
```

Comment:
You should not mock anything in systemUnderTest. Basically you are testing Mockito here.